### PR TITLE
#162159462 Create content placeholder loaders on dashboard

### PR DESCRIPTION
**Pull request by DennisWanjiru**
#### What does this PR do?
create data placeholders to represent the expected data appearance

#### Description of Task to be completed?
- Replace all the spinners on the dashboard with content placeholder loaders 

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
`ft-dashboard-content-loader-162159462`
- Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super[]Administrator(10948) 

#### What are the relevant pivotal tracker stories?
[#162159462](https://www.pivotaltracker.com/story/show/162159462)

#### Screenshots (if appropriate)
ezgif com-crop 4

### CODE REVIEW

File: src/components/Placeholders/__tests__/AnalyticsCardPlaceholder.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AnalyticsCardPlaceholder from '../AnalyticsCardPlaceholder';
+
+
+describe('<AnalyticsCardPlaceholder />', () => {
+  const wrapper = shallow(<AnalyticsCardPlaceholder />);
+
+  it('should render without crushing', () => {
```
Comment (CEOehis) [inline src/components/Placeholders/__tests__/AnalyticsCardPlaceholder.test.js]: Typo: `crushing`. The same typo is on the other test files in this folder.

Comment (DennisWanjiru) [reply src/components/Placeholders/__tests__/AnalyticsCardPlaceholder.test.js]: okay on it

Comment (DennisWanjiru) [reply src/components/Placeholders/__tests__/AnalyticsCardPlaceholder.test.js]: @CEOehis done

File: src/views/Analytics/index.scss [hunk 2]
```diff
@@ -5,3 +5,20 @@
...
+.analytics-card {
+  height: 225px;
+  padding: 0 30px 30px 30px;
+  font-size: 15px;
+  color: #4F4F4F;
+  line-height: 20px;
+  box-sizing: border-box;
+  font-family: "DIN Pro";
+  background-color: #FFFFFF;
+  width: 340px;	border-radius: 2px;
```
[the remark this answers is not in the document]
Comment (DennisWanjiru) [reply src/views/Analytics/index.scss]: Ohh yeah thanks for the highlight

**Comment by DennisWanjiru** [comment]
@amarachukwu-agbo Done

**Comment by amarachukwu-agbo** [comment]
LGTM

**Pull request merged**